value is too large to be recorded on hdrhistogram

```
2022/09/29 20:44:52 Using a total of 64 concurrent benchmark workers
2022/09/29 20:44:52 Using input file to produce terms for the benchmarks
2022/09/29 20:44:53 Starting term-level queries benchmark: Type SUFFIX
                 Test time                      Command Rate   Client p50 with RTT(ms)            Total Commands
panic: value 1055646 is too large to be recorded       80.97                   494.591                        81

goroutine 114 [running]:
main.Benchmark.func1()
	/github/workspace/benchmark.go:177 +0x20e
created by main.Benchmark

```

--- a/benchmark.go
+++ b/benchmark.go
@@ -140,7 +140,7 @@ func WildcardBenchmark(terms []string, field string, idx index.Index, prefixMinL
 //
 // If outfile is "-" we write the result to stdout
 func Benchmark(concurrency int, duration time.Duration, instantMutex *sync.Mutex, engine, title string, outfile string, reportingPeriod time.Duration, tab *tabwriter.Writer, f func() error) {
-	totalHistogram = hdrhistogram.New(1, 1000000, 3)
+	totalHistogram = hdrhistogram.New(1, 1000000000, 3)
 
 	var out io.WriteCloser
 	var err error

